app/src/main/kotlin/com/intive/picover/parties/state/PartiesState.kt

```diff
@@ -5,4 +5,5 @@ import com.intive.picover.parties.model.Party
 sealed class PartiesState {
```

Comment:
@marcinWisnia you will adjust it to use base state from your other merge request after merge?